src/physics/physics.h

```diff
@@ -11,112 +16,136 @@ struct Box {
     real mass;
     real curr_t;
 
+    ///* integrate_part
+    // Integrates part of a whole step.
     void integrate_part(real t, real delta) {
         position += velocity * t * delta;
         curr_t += t;
         ASSERT_LT(curr_t, 1.0);
     }
 
+    ///* integrate
+    // Integrates the rest of the time step.
     void integrate(real delta) {
         position += velocity * (1 - curr_t) * delta;
         curr_t = 0;
     }
 
-    Box extend(const Vec3 &ext) const {
-        Box copy = *this;
+    ///* extend
+    // Extends the domain of this AABody
+    // with the vector passed in.
+    AABody extend(const Vec3 &ext) const {
+        AABody copy = *this;
         copy.half_size += ext;
         return copy;
     }
 };
 
-struct RayHit {
+///* Manifold
+// An object with information about the collision.
+struct Manifold {
```

Comment:
You can write explicit titles if you want, it doesn't really matter, but the documentation generator can parse struct names if you want.